src/app/nhs-app-template-submitted/[templateId]/page.tsx

```diff
@@ -1,8 +1,13 @@
 'use server';
 
-const NhsAppTemplateSubmittedPage = async () => (
+const NhsAppTemplateSubmittedPage = async ({
+  params,
+}: {
+  params: { templateId: string };
+}) => (
   <h2 className='nhsuk-heading-l' data-testid='page-sub-heading'>
     Placeholder page
+    <p>{params.templateId}</p>
```

Comment:
As per previous discussion, lets not put unsanitized user data directly into the page.

Reply:
updated. Good shout, thanks.